app/main.py

```diff
@@ -1,3 +1,79 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance") -> "Distance":
```

Comment:
```suggestion
    def __add__(self, other: Distance | int | float) -> Distance:
```

Reply:
Traceback (most recent call last):
  File "D:\ProgramPyCharm\Distance class\py-distance-class\tests\test_main.py", line 6, in <module>
    from app.main import Distance
  File "D:\ProgramPyCharm\Distance class\py-distance-class\app\main.py", line 2, in <module>
    class Distance:
  File "D:\ProgramPyCharm\Distance class\py-distance-class\app\main.py", line 12, in Distance
    def __add__(self, other: "Distance" | int | float) -> "Distance":
                             ~~~~~~~~~~~^~~~~
TypeError: unsupported operand type(s) for |: 'str' and 'type'

Process finished with exit code 1



here is such an error, I don't understand what the problem